src/libraries/System.IO.Pipelines/src/System/IO/Pipelines/PipeReader.cs

```diff
@@ -119,6 +119,16 @@ public static PipeReader Create(Stream stream, StreamPipeReaderOptions? readerOp
             return new StreamPipeReader(stream, readerOptions ?? StreamPipeReaderOptions.s_default);
         }
 
+        /// <summary>
+        /// Creates a <see cref="PipeReader"/> wrapping the specified <see cref="ReadOnlySequence{T}"/>.
```

Comment:
Is this fine? It points to the docs for `ReadOnlySequence<T>`, with no indication of it being only for bytes. Is this just how things are?

Reply:
Is that how the other docs are?

Reply:
I tried to google this issue (because you can't do `ReadOnlySequence<byte>`), and the idea here is that there is no docs page to link to for `ReadOnlySequence<byte>`, but the is for `ReadOnlySequence<T>`, so that is why I believe. I'm not sure where else I would find docs like this in .NET as I can't recall any off the top of my head at least.